sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_Logging.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It would be good to move the removed samples to Azure.Core samples. All the removed samples illustrate Azure.Core concepts.

Reply:
@pakrym is going to add these to Azure.Core separately.